### PR TITLE
LPS-160222 | master

### PR DESCRIPTION
**Pull request by sontruongces**
Hi team,

This is the PR for the [LPS-160222](https://issues.liferay.com/browse/LPS-160222), the original has passed my review at sontruongces#45. Please help me to review and give feedback.

Thanks!
cc: @TaiPhamLR

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/LocalizedItemSelectorRendering.java [hunk 1]
```diff
@@ -59,7 +59,15 @@ public void add(ItemSelectorViewRenderer itemSelectorViewRenderer) {
...
-		_itemSelectorViewRenderers.put(title, itemSelectorViewRenderer);
+		if (_itemSelectorViewRenderers.put(title, itemSelectorViewRenderer) !=
+				null) {
+
+			_navigationItems.removeIf(
+				navigationItem -> title.equals(
+					navigationItem.get(
+						"label"
+					).toString()));
+		}
```
Comment (adolfopa) [inline modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/LocalizedItemSelectorRendering.java:70]: Can you split the update of the data structure from the check?
```
ItemSelectorViewRenderer previousItemSelectorViewRenderer = _itemSelectorViewRenderers.put(...);

if (previousItemSelectorViewRenderer != null) {
    ...
}

Comment (adolfopa) [reply modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/LocalizedItemSelectorRendering.java:70]: Also, avoid chaining method calls. In
```
					navigationItem.get(
						"label"
					).toString()
```
replace the toString() call with `String.valueOf(navigationItem.get("label"))`.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by TaiPhamLR** [comment]
Hi @adolfopa,
Thanks for your review. I fixed the code as per your kind suggestion.

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**